data.json

```
@@ -760,5 +760,11 @@
     "githubUsername": "anotherwebguy",
     "imageUrl": "https://user-images.githubusercontent.com/66346161/114270825-18c90d80-9a2c-11eb-9173-1c072e90ac27.png",
     "issueNumber": 436
+  },
+   {
```

```suggestion
  {
```
please make this line up with the other braces